i18n/en-US.json

```diff
@@ -283,5 +283,40 @@
 
 	"__comment": "UI-1916: Add app specific shortcut",
 	"__version": "4.0",
-	"toggleAll": "Toggle All"
+	"toggleAll": "Toggle All",
+
+	"newAccountWizard": {
```

Comment:
For new apps, we always add a root node (like so `<appName>App`) to avoid path collisions.

In this case, this app is not new but this submodule access path should look like this:
```
accountsApp.wizard
```

Reply:
Are i18n files merged into a single structure? Are not bounded to the current app only?

Reply:
The app i18n file is merged with the core i18n file, that's why you can access simple strings like `i18n.yes`, `i18n.no` or `i18n.search`.